src/content/questionnaire/ListView.test.tsx

```diff
@@ -0,0 +1,566 @@
+  dataCommons: [],
+  createdAt: "",
+  updateAt: "",
+  studies: null,
+};
+
+const defaultMocks: MockedResponse[] = [
+  {
+    request: {
+      query: LIST_APPLICATIONS,
+      variables: {
+        first: 20,
+        offset: 0,
+        sortDirection: "desc",
+        orderBy: "submittedDate",
+      },
+      context: { clientName: "backend" },
+    },
+    result: {
+      data: {
+        listApplications: {
+          total: 0,
+          applications: [],
+        },
+      },
+    },
+  },
+];
+
+type ParentProps = {
+  mocks?: MockedResponse[];
+  initialEntries?: MemoryRouterProps["initialEntries"];
+  role?: UserRole;
+  children: React.ReactNode;
+};
+
+const TestParent: FC<ParentProps> = ({
+  mocks = defaultMocks,
+  initialEntries = ["/"],
+  role = "Submitter",
+  children,
+}: ParentProps) => {
+  const baseAuthCtx: AuthContextState = useMemo<AuthContextState>(
+    () => ({
+      status: AuthStatus.LOADED,
+      isLoggedIn: role !== null,
+      user: { ...baseUser, role },
+    }),
+    [role]
+  );
+
+  return (
+    <MockedProvider mocks={mocks} addTypename={false}>
+      <MemoryRouter initialEntries={initialEntries}>
+        <AuthContext.Provider value={baseAuthCtx}>
+          <SearchParamsProvider>{children}</SearchParamsProvider>
+        </AuthContext.Provider>
+      </MemoryRouter>
+    </MockedProvider>
+  );
+};
+
+describe("Accessibility", () => {
+  it("has no accessibility violations", async () => {
+    const { container } = render(
+      <TestParent>
+        <ListView />
+      </TestParent>
+    );
+
+    await waitFor(async () => {
+      const results = await axe(container);
+      expect(results).toHaveNoViolations();
+    });
+  });
+});
+
+describe("ListView Component", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    jest.useRealTimers();
+  });
+
+  it("renders without crashing", () => {
+    const { getByText } = render(
+      <TestParent>
+        <ListView />
+      </TestParent>
+    );
+    expect(getByText("Submission Request List")).toBeInTheDocument();
+  });
+
+  it("sets the page title correctly", () => {
+    render(
+      <TestParent>
+        <ListView />
+      </TestParent>
+    );
+    expect(mockUsePageTitle).toHaveBeenCalledWith("Submission Request List");
+  });
+
+  it.each<UserRole>(["User", "Submitter", "Organization Owner"])(
+    "shows the 'Start a Submission Request' button for '%s' role",
+    (role) => {
+      const { getByText } = render(
+        <TestParent role={role}>
+          <ListView />
+        </TestParent>
+      );
+      expect(getByText("Start a Submission Request")).toBeInTheDocument();
+    }
+  );
+
+  it.each<UserRole>([
+    "Admin",
+    "Data Commons POC",
+    "Data Curator",
+    "Federal Lead",
+    "Federal Monitor",
+    "fake-role" as UserRole,
+  ])("should not show the 'Start a Submission Request' button for '%s' role", (role) => {
+    const { queryByText } = render(
+      <TestParent role={"fake-role" as UserRole}>
```

Comment:
```suggestion
      <TestParent role={role}>
```

Reply:
Oops, thanks for catching this. Fixed in [c3a8a83](https://github.com/CBIIT/crdc-datahub-ui/pull/537/commits/c3a8a83dd960125f168015736563f6a8c3285973).